MobileClient/src/providers/background-tracker-service.ts

```diff
@@ -57,6 +57,27 @@ export class BackgroundTrackerService {
       return this.startPromise;
     }
 
+    let locationStatus;
```

Comment:
Resolve the promise with the location status instead of having it in a separate variable.